ballerina/json_api.bal

```diff
@@ -62,9 +62,10 @@ public isolated function toJson(anydata v)
 # Represent the options that can be used to modify the behaviour of the projection.
 #
 # + allowDataProjection - enable or disable projection
-public type Options record {|
-    boolean allowDataProjection = true;
-|};
+public type Options record {
+    boolean nilAsOptionalField = false;
+    boolean absentAsNilableType = false;
+}|false;
 
```

Comment:
This may not be the correct config. `Options options = false` doesn't capture the right semantics. 

I was suggesting something like this

```ballerina
type Options record {
    record {
        boolean nilAsOptionalField = false;
        boolean absentAsNilableType = false;
    }|false allowDataProjection = {}
};
```

Reply:
Got it. I will update.

Reply:
Updated https://github.com/ballerina-platform/module-ballerina-data.jsondata/pull/4/commits/9ec1cbc313e5bfdf4aa370a4d8dc4091a0257dc8